sky/setup_files/setup.py

```diff
@@ -83,6 +83,7 @@ def parse_readme(readme: str) -> str:
     'ray[default]>=1.9.0,<=2.2.0',
     'rich',
     'tabulate',
+    'typing-extensions',
```

Comment:
Can we remove our dependency? Like L31 and L32 in storage.py, can we change L33 and L34 to:
Before
```
StorageHandle = Any
StorageStatus = global_user_state.StorageStatus
Path: TypeAlias = str
SourceType: TypeAlias = Union[Path, List[Path]]
```

After
```
StorageHandle = Any
StorageStatus = global_user_state.StorageStatus
Path = str
SourceType = Union[Path, List[Path]]
```

Reply:
mypy (format.py) appears to pass with this change

Reply:
Ahh, great catch! Removed the dependency. Thanks!